packages/ember-routing/lib/system/router.js

```diff
@@ -609,7 +609,6 @@ const EmberRouter = EmberObject.extend(Evented, {
       }
 
       handler._setRouteName(routeName);
-      handler._populateQPMeta();
```

Comment:
@scalvert - Great question, general reasons below:

* if you look at `_populateQPMeta()`'s definition, all it does is eagerly `this.get('_qp')` and save it off into the bucket cache, however *nothing ever looks it up*!!!
* Things like `Ember.Router#_prepareQueryParams` (used by `{{link-to`, `transitionTo`, etc) are calling `route._getQPMeta()` which does `return route.get('_qp')` (I'm guessing that at some point it was returning from the bucket cache, but it does not do this any longer).  This still works (regardless of `_populateQPMeta()` being called first).